components/iscesys/ImageApi/DataAccessor/bindings/DataAccessormodule.cpp

```diff
@@ -136,8 +136,17 @@ createAccessor_C(PyObject* self, PyObject* args)
   }
   if (casterCh[0] == '\0')
   {
-    ptDataAccessor = (uint64_t) AF->createAccessor(filename, filemode, size,
-        bands, width, scheme);
+    try
+    {
+      ptDataAccessor = (uint64_t) AF->createAccessor(filename, filemode, size,
+          bands, width, scheme);
+    }
+    catch(const std::exception& e)
+    {
+      PyErr_SetString(PyExc_OSError, e.what());
```

Comment:
In the future we may wish to distinguish between e.g. FileNotFoundError/PermissionError/IsADirectoryError/etc. For now this looks good

Reply:
`CPLGetLastErrorMsg` might be handy to catch the exact error.

Reply:
Oh yep, I can add that either in addition to/instead of the manual `errMsg` that I'm passing